src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_shipping.html.twig

```diff
@@ -151,10 +151,7 @@
 </div>
 
 <div class="col-md-12">
-  <div id="warehouse_combination_collection" class="col-md-12 form-group" data-url="{{ path('admin_warehouse_refresh_product_warehouse_combination_form') }}">
-    {% if asm_globally_activated and isNotVirtual and isChecked %}
-      {{ include('@PrestaShop/Admin:Product/ProductPage/Forms/form_warehouse_combination.html.twig', { 'warehouses': warehouses, 'form': form }) }}
-    {% endif %}
+  <div id="warehouse_combination_collection" class="col-md-12 form-group">
```

Comment:
Why not remove the complete `div` ?

Reply:
Wouldn't it be a BC Break then?

Reply:
Templates from BO isn't part from our Public API (just seen with @eternoendless), otherwise, each time we change anything in a template is a bc breaks :sweat_smile: